test/models/Offer/Model.test.ts

```diff
@@ -22,6 +24,18 @@ describe("Offer", () => {
     await expect(offer.validate()).resolves.not.toThrow();
   });
 
+  it("should create a valid offer with default extensionApprovalStatus", async () => {
```

Comment:
Le sacaría el `should`:
`creates a valid offer with default extensionApprovalStatus`